.github/workflows/test.yml

```diff
@@ -25,19 +24,30 @@ jobs:
         - name: Install
           run: npm install
         # - name: Download wallet
-        #   run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v${{ matrix.vegawallet-version }}/vegawallet-linux-amd64.zip -O
+        #   run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v0.13.1/vegawallet-linux-amd64.zip -O
+        #   # run: curl -L https://github.com/vegaprotocol/vegawallet/releases/download/v${{ matrix.vegawallet-version }}/vegawallet-linux-amd64.zip -O
+        # - name: ls current
+        #   run: ls
+        # - name: ls previous
+        #   run: ls ..
         # - name: Unzip wallet
-        #   run: unzip ./vegawallet-linux-amd64.zip
+        #   run: unzip vegawallet-linux-amd64.zip
+        - name: Download and Unzip latest wallet
+          run: curl -s https://api.github.com/repos/vegaprotocol/vegawallet/releases/latest | awk -F\" '/browser_download_url.*linux-amd64.zip/{print $(NF-1)}' | xargs wget && unzip vegawallet-linux-amd64.zip && rm vegawallet-linux-amd64.zip
         - name: Initialize wallet
           run: ./vegawallet init -f
-        # - name: List keys
-        #   run: ./vegawallet key list 
+        - name: Create recovery phrase file
+          run: echo "feel aisle depend tower spell loan benefit stock office usual gallery kick mechanic tumble dad poem meat marine knock call border dismiss permit nominee" > recovery-phrase.txt
```

Comment:
**nitpick (blocking)** Let's move this to an environment variable so we can store it in a secret. I mean, the secret is already out, but we might want to change it in future